chat/prekey/key.go

```diff
@@ -119,3 +119,8 @@ func FromProtoBuf(preKey pb.PreKey) (PreKey, error) {
 	copy(p.identityPublicKey[:], preKey.IdentityKey[:32])
 	return p, nil
 }
+
+// check if pre key is older than given date
+func (p PreKey) OlderThen(past time.Duration) bool {
```

Comment:
OlderThen should be renamed to OlderThan to be grammatically correct.

Reply:
done